usr/bin/torbrowser

```diff
@@ -433,9 +465,10 @@ Then try again.</p>'
 
 main_function() {
    root_check "$@"
-   tb_preparation "$@"
```

Comment:
Somehow the discussion on this one vanished. Perhaps due to newer revision. Just as placeholder.

Reply:
It looks like it will also make generating a set of profiles for the various zeronet scenarios easier, because the variables I added can be used to configure alternative browsers from scripts like /usr/bin/i2pbrowser does.

Reply:
> I think the best thing to do would be to completely remove the --i2p flag from /usr/bin/torbrowser and set them all with environment variables in /usr/bin/i2pbrowser instead.

That sounds great!